index.html

```diff
@@ -11,12 +11,12 @@
         <link href="https://fonts.googleapis.com/css?family=RobotoDraft:400,300,500,700,400italic" rel="stylesheet" >
         <link href='http://fonts.googleapis.com/css?family=Open+Sans:300italic,700italic,300,700' rel='stylesheet'>
         <!-- bower:css -->
-        <link rel="stylesheet" href="https://ajax.googleapis.com/ajax/libs/angular_material/1.0.0-rc3/angular-material.min.css">
-        <link rel="stylesheet" href="//maxcdn.bootstrapcdn.com/font-awesome/4.3.0/css/font-awesome.min.css">
-        <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
+        <link rel="stylesheet" href="/bower_components/angular-material/angular-material.css">
+        <link rel="stylesheet" href="/bower_components/google-material-icons/dist/material-icons-font.css">
+        <link rel="stylesheet" href="/bower_components/font-awesome/css/font-awesome.css">
```

Comment:
On running `gulp prod`, it gives error saying [line 8](https://github.com/gdg-x/boomerang/pull/79/files#diff-6d6f6357791623581a22d402b2c54e17R8) is too long